Shared/da.lproj/Localizable.strings

```diff
@@ -1588,28 +1648,25 @@
 "Settings.Home.Option.Wallpaper" = "Baggrund";
 
 /* In the settings menu, on the Firefox wallpaper customization screen, this is the accessibility string for the amethyst firefox wallpaper. */
-"Settings.Home.Option.Wallpaper.Accessibility.AmethystWallpaper" = "Firefox-baggrund, ametyst-mønster, knap";
+"Settings.Home.Option.Wallpaper.Accessibility.AmethystWallpaper.v99" = "Firefox-baggrund, ametyst-mønster.";
```

Comment:
some strings have .v99 in them as long as that's okay we are good

Reply:
Yeah it was an easy way to export strings for translation without having to add `v2` in the name. Ideally this should be automated but I was not successful on that end.